src/sfizz/Region.cpp

```diff
@@ -381,6 +393,226 @@ bool sfz::Region::parseOpcode(const Opcode& opcode)
         setValueFromOpcode(opcode, rtDecay, Default::rtDecayRange);
         break;
 
+    // Performance parameters: filters
+    case hash("cutoff"):
+        {
+            const auto filterIndex { backParameter.value_or(1) - 1 };
+            if (!extendIfNecessary(filters, filterIndex + 1))
```

Comment:
Can we these vectors be reserved with some reasonable initial capacity?

Reply:
Hm, yes, but then it would mean all regions with no EQ/filter would allocate space for them. The reasonable choices would probably be 2 for filters and 3 for EQs, since these are the SFZ v1 defaults.

On possibility would be to reserve upon first seeing a fitler/eq opcode, so as to have only 1 allocation. I updated doing this.